src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -249,7 +249,7 @@ class HaConfigHardware extends SubscribeMixin(LitElement) {
                         .twoline=${Boolean(boardId)}
                       >
                         ${imageURL
-                          ? html`<img slot="graphic" src=${imageURL} />`
+                          ? html`<img alt="" slot="graphic" src=${imageURL} />`
```

Comment:
There is additional text information around this image, therefore I believe the image to be decorative.